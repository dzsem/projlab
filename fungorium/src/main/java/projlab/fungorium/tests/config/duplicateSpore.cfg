# Duplicate Spore
# ------------------------------------------
# Init state
# ------------------------------------------
SET TECTONSPLITCHANCE 0

# 0
ADD TECTON
# 1
ADD MUSHROOMSPORE 0
SET EFFECTGENERATION 1 DUPLICATE
# 2
ADD INSECT 1 0