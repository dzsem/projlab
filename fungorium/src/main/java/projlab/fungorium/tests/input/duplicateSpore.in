# Duplicate Spore
# ------------------------------------------
# Input
# ------------------------------------------
# Duplicate insect
# 3
LOAD config/duplicateSpore.cfg
INSECT 2 EATSPORE
# Print states
# ------------------------------------------
SAVE duplicateSpore.out