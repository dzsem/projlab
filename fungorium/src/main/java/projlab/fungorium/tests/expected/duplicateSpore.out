INSECT 2 0 1 1 0
INSECT 3 0 1 1 0